tests/gcp/operators/test_cloud_memorystore_system_helper.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""
+Helpers to perform system tests for the Google Cloud Build service.
+"""
+import argparse
+import os
+from urllib.parse import urlparse
+
+from tests.contrib.utils.gcp_authenticator import GcpAuthenticator, GCP_GCS_TRANSFER_KEY # TODO: Update it
+from tests.contrib.utils.logging_command_executor import LoggingCommandExecutor
+
+GCP_PROJECT_ID = os.environ.get("GCP_PROJECT_ID", "example-project")
+GCP_ARCHIVE_URL = os.environ.get('GCP_MEMORYSTORE_EXPORT_GCS_URL', 'gs://test-memorystore/my-export.rdb')
+GCP_ARCHIVE_URL_PARTS = urlparse(GCP_ARCHIVE_URL)
+GCP_BUCKET_NAME = GCP_ARCHIVE_URL_PARTS.netloc
+GCP_OBJECT_NAME = GCP_ARCHIVE_URL_PARTS.path[1:]
```

Comment:
I am wondering if it would be a good idea to import those constants from the example DAG? It would be safer in case of any change.

Reply:
We discussed this today ✅

Reply:
I think, it's good idea, but in the past, we did not import example_dag from test, because it execute a database query during initialization.